### ISSUE TITLE
Signal keeps crashing when PayPal password is entered (via Passwort Manager)

### ISSUE THREAD

**Issue by Linkinsoldier** [signalapp/Signal-Android#13862]
### Guidelines

- [X] I have searched searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] This isn't a feature request or a discussion topic

### Bug description

Preconditions:

Go to donate in the app, 
select desired one time money value
Select PayPal as payment option
Insert email address

Expected behaviour:
Passwort can be inserted (via password manager)

Seen behaviour:
The moment I use auto fill in my keepass password manager or even when I paste it via copy and paste, the signal app crashes.

Just to be sure: it does not crash when I try to login, but the moment I insert the Password!

Version:
Latest Google play app release 7.28.4.

Device:
Google Pixel 9 Pro

Android version:
Latest Android 15

### Device

Pixel 9 Pro

### Android version

15 (latest)

### Signal version

7.28.4.

**Comment by Sagar0-0** [comment]
The logs says, you are entering the Email address in the Custom Amount field, which does not makes sense, can you please share a screen recording for better understanding of the issue.(Make sure to hide your credentials)

**Comment by Linkinsoldier** [comment]
I don't know how to do this. 

But now it gets very interesting: before I created the ticket (and once when I tried to make the screen capture) I always had to enter the email address on one screen first, then I get another screen to enter the password and there it fails. 
Then I played around with pay via g pay and credit card, go back into pay with PayPal and there It now gives me the option to insert password and email on the same screen (not 2 screens). Now I could insert and login... But now I can no longer reproduce it because I am always logges in (maybe need to wait).

PS: I also changed my password after the debug log... Was a bit too scared what might be contained in it...

**Comment by Sagar0-0** [comment]
> Was a bit too scared what might be contained in it...

Don't worry your email and password are not exposed.

> But I definitely selected a proposed amount (10 or 25 bucks)

If this is the case then should have shown boost buttons instead of Custom Amount, so what I think is...

> The password via keypass on the third screen also enters the email address on the first screen

This is the situation that triggered the email to be set on the first "Custom Amount" Screen. and caused the crash.

I will try to reproduce this issue, and if I succeed then update it here n solve it. Thanks for raising the ticket.

**Comment by Linkinsoldier** [comment]
thanks for the reponse. I added a screenshot of the third screen (where supposedly only the password is entered before crashing. I hope that helps?

there are 6 buttons for me, i select one of them and then proceed with "continue" (then the custom field is gone), then email, and then the password to be entered on the third screen.

**Comment by Linkinsoldier** [comment]
> I will try to reproduce this issue, and if I succeed then update it here n solve it. Thanks for raising the ticket.

then i hope this helps:
I use KeepassDX as Password Manager on Android and i am trying to use firefox as browser (not sure what Signal uses internally)

**Comment by alex-signal** [comment]
Signal uses a WebView here which I *believe* uses webkit internally. It's definitely possible that the Custom Amount field is focused or something like that, since we are displaying a window overtop of it. 

For what its worth, I can't reproduce this issue using bitwarden.

**Comment by Sagar0-0** [comment]
Yes, Google password manager won't reproduce this issue for me too.
